Index: include/libnest2d/backends/clipper/clipper_polygon.hpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/include/libnest2d/backends/clipper/clipper_polygon.hpp b/include/libnest2d/backends/clipper/clipper_polygon.hpp
--- a/include/libnest2d/backends/clipper/clipper_polygon.hpp	(revision 031e4e61218edd092fbc54fbc6df145287b628ba)
+++ b/include/libnest2d/backends/clipper/clipper_polygon.hpp	(date 1627556142178)
@@ -1,7 +1,7 @@
 #ifndef CLIPPER_POLYGON_HPP
 #define CLIPPER_POLYGON_HPP
 
-#include <clipper.hpp>
+#include <polyclipping/clipper.hpp>
 
 namespace ClipperLib {
 
